src/Tools/ExternalAccess/FSharp/Navigation/FSharpDocumentNavigationService.cs

```diff
@@ -50,31 +53,34 @@ public bool CanNavigateToPosition(Workspace workspace, DocumentId documentId, in
             return service.CanNavigateToPosition(workspace, documentId, position, virtualSpace, cancellationToken);
         }
 
+        [Obsolete("Call overload that takes a CancellationToken", error: false)]
         public bool TryNavigateToSpan(Workspace workspace, DocumentId documentId, TextSpan textSpan, OptionSet options)
-            => TryNavigateToSpan(workspace, documentId, textSpan, options, CancellationToken.None);
+            => TryNavigateToSpan(workspace, documentId, textSpan, CancellationToken.None);
 
-        public bool TryNavigateToSpan(Workspace workspace, DocumentId documentId, TextSpan textSpan, OptionSet options, CancellationToken cancellationToken)
+        public bool TryNavigateToSpan(Workspace workspace, DocumentId documentId, TextSpan textSpan, CancellationToken cancellationToken)
         {
             var service = workspace.Services.GetService<IDocumentNavigationService>();
-            return service.TryNavigateToSpan(workspace, documentId, textSpan, options, cancellationToken);
+            return service.TryNavigateToSpan(workspace, documentId, textSpan, NavigationOptions.Default with { PreferProvisionalTab = true }, cancellationToken);
```

Comment:
Why are we setting `PreferProvisionalTab` to `true` for F#? Are they currently using the OptionSet to control this behavior?

Reply:
Yes, they set it to true.